platform/android/java/editor/src/main/java/org/godotengine/editor/GodotEditor.kt

```diff
@@ -189,6 +249,31 @@ open class GodotEditor : GodotActivity() {
 		return instanceId
 	}
 
+/* TODO - FROM XR EDITOR - don't know how to combine this with the one added in master
```

Comment:
@m4gr3d I didn't know how to merge these two different versions of the `onNewGodotInstanceRequested` function, so you'll need to fix this up :)